sdk/eventgrid/eventgrid-namespaces/test/public/utils/testUtils.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * A constant that indicates whether the environment is node.js or browser based.
+ */
+export const isNode =
```

Comment:
should we use `isNode` from core-util now?

Reply:
Removed it (in next commit)